src/app/Main/Body/QueryPage/QueryPage-Container.component.tsx

```diff
@@ -64,15 +69,28 @@ export function QueryPageContainerComponent(): JSX.Element {
                 uniqueValuesObject[id] = filterElement.values;
             });
 
+            const nrOfSelectedIsolates = isolateCountProp.totalNumberOfIsolates;
+
             setData({
+                ...data,

```

Comment:
? is there some other data that is not setted here?

Reply:
Done. Commit "Make ist simple"